Documentation/helm-values.rst

```diff
@@ -90,6 +90,14 @@
      - Seconds after which the completed job pod will be deleted
      - int
      - ``1800``
+   * - cgroup
+     - Configure cgroup related configuration
+     - object
+     - ``{"hostRoot":"/run/cilium/cgroupv2"}``
+   * - cgroup.hostRoot
+     - Configure cgroup root where cgroup2 filesystem is mounted on the host (see also: `cgroup.autoMount`)
```

Comment:
nit: if going patch-by-patch, the:

> (see also: `cgroup.autoMount`)

is added in a prior patch than the autoMount option.

Not blocking on my side.